create_mattes_from_colored_mattes.py

```diff
@@ -252,8 +252,12 @@ def create_frame_dict(files):
         matte_colors = get_matte_colors(image)    
 
         for element in bw_matte_elements:
-            element_matte_path = rf'{matte_path}\{element}'
-            element_color = matte_element_to_color[element]
+            # element_matte_path = rf'{matte_path}\{element}'
+            # element is now part of matte_path
+            element_matte_path = rf'{matte_path}'
+            # element_color = matte_element_to_color[element]
+            # Always look for 'dorothy red' now
+            element_color = matte_element_to_color['dorothy']
```

Comment:
we might want to be careful about submitting anything that is project specific here

Reply:
Yeah, I was doing the quickest fix. I think it's better just to supply the color, which will also remove the dependency on the csv file.